### PR TITLE
Remove use of __EVAL macro

### PR DESCRIPTION
**Pull request by PabstMirror**
HEMTT does not understand this macro
this feel very dirty, but it works

### CODE REVIEW

**Review by mharis001 — APPROVED** [review]

**Review by kymckay — APPROVED** [review]

**Review by jonpas — APPROVED** [review]
This actually makes more sense if you look at it from a preprocessing view.

File: addons/ui/RscDisplayOptionsLayout.hpp [hunk 1]
```diff
@@ -12,7 +12,7 @@ class RscDisplayOptionsLayout {
             onMouseButtonUp = "with uiNamespace do {['mouseButtonUp',_this,''] call RscDisplayOptionsLayout_script;};";
         };
         #define ADD_ELEMENT(var1) class Element##var1: Element022 {\
-            idc = __EVAL(12000 + var1);\
+            idc = 12##var1;\
```
Comment (commy2) [inline addons/ui/RscDisplayOptionsLayout.hpp:15]:
```suggestion
            idc = QUOTE(12000 + (var1));\
```

Comment (PabstMirror) [reply addons/ui/RscDisplayOptionsLayout.hpp:15]: right now it's a number which is ideal
but `1+1 = 11` type "addition" is weird and hard to understand
so I'm fine making it a statement that needs to be evaluated

Comment (commy2) [reply addons/ui/RscDisplayOptionsLayout.hpp:15]: It's a plus b. What's not to understand? This addresses the issue while keeping the original numbers.

Comment (PabstMirror) [reply addons/ui/RscDisplayOptionsLayout.hpp:15]: (12##023) = 12023 = (12000 + 023)

**Pull request merged**